src/lib/homestore_backend/heap_chunk_selector.cpp

```diff
@@ -293,7 +296,10 @@ std::optional< homestore::chunk_num_t > HeapChunkSelector::get_most_available_bl
                              if (a->available() && !b->available()) { return false; }
                              return false;
                          });
-
+    if (!(*max_it)->available()) {
```

Comment:
@xiaoxichen  check if have available chunk here.
 And this pr make sure creating an empty PG is not supported, so we neednt check if `m_pg_chunks` is empty.